core/arch/aarch64/aarch64.asm

```diff
@@ -104,19 +104,6 @@ DECL_EXTERN(initstack_mutex)
 DECL_EXTERN(icache_op_struct)
```

Comment:
> On aarch64 conditional branch instructions have 19 bits of offset and unconditional branch and call instructions have 26 bits, and is used for every other function call so we should be good for awhile.

grammar: s/is used/are used/; s/awhile/a while/

Reply:
Done